app/src/view/screens/Sell.js

```
@@ -204,74 +205,72 @@ export default function Sell({navigation}) {
   }, [isFocused, authToken]);
 
   const fetchCategory = async (token) => {
- 
-
     try {
       const response = await fetch(
-        base_url + 'itemCategory/getAllItemCategories?page=1&limit=10000',
+        base_url + "itemCategory/getAllItemCategories?page=1&limit=10000",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)